Examples/rc-maestro/mise.toml

```diff
@@ -0,0 +1,2 @@
+[tools]
```

Comment:
Do you plan to use different Tuist versions throughout the repo? If not, I'd consider moving this one to the root.